### PR TITLE
WP-868: add support for updating the plugin through WordPress plugin management page

### PR DESCRIPTION
**Pull request by swang-va**
@vendasta/colonelpanic

### CODE REVIEW

File: business-profile-render.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?php
 /**
- * Plugin Name: Business Profile Data
- * Plugin URI: /wp-admin/tools.php?page=businessprofiledata
```
Comment (mzhao-va) [inline business-profile-render.php:4]: May want both urls so that user know that the plugin description/how-to-use guide lives under tools tab?

**Review by mzhao-va — APPROVED** [review]
Make sense to me although no idea if it'd work 🤷 
How do we test it?

File: business-profile-render.php (hunk 1 above)
Comment (swang-va) [reply business-profile-render.php:4]: This URL has its dedicated purpose. If you search for `$plugin['PluginURI']` in this PR you would see it is actually used by the plugin information page

**Comment by swang-va** [comment]
> Make sense to me although no idea if it'd work 🤷
> How do we test it?

I copied it to a local site that I had, changed the version number to a lower version manually, and then WordPress prompted for the update.

**Pull request merged**